WcaOnRails/Gemfile.lock

```diff
@@ -724,4 +724,4 @@ DEPENDENCIES
   wicked_pdf
 
 BUNDLED WITH
-   2.1.0
+   2.1.4
```

Comment:
@jonatanklosko are we fine with this?
It would obviously be good that production is up to date, but we may have to run chef there to update bundler to this version.

Reply:
Oops, that was not intentional! It's probably a good idea to update but that probably belongs in a separate commit. Should I remove it?

Reply:
Actually this wouldn't prevent any gem updates, would just produce a warning that the current version is older than the one in `Gemfile.lock`. I think we used to bump that from time to time without paying much attention ^^

Reply:
It looks like the only changes between the versions are bug fixes: https://github.com/rubygems/bundler/blob/master/CHANGELOG.md